mmv1/third_party/terraform/go.mod.erb

```diff
@@ -3,7 +3,7 @@ module github.com/hashicorp/terraform-provider-google<%= "-" + version unless ve
 
 require (
 	cloud.google.com/go/bigtable v1.10.1
-	github.com/GoogleCloudPlatform/declarative-resource-client-library v0.0.0-20211027225138-ef28ca390518
+	github.com/GoogleCloudPlatform/declarative-resource-client-library v0.0.0-20211111235242-eee3ce55868a
```

Comment:
Note, we'll want to handwrite documentation in the DCL and add reference links such as was done in https://github.com/GoogleCloudPlatform/magic-modules/pull/5325. That'll put the generated docs are at parity with the original ones. See the current diff:

![image](https://user-images.githubusercontent.com/2080520/141513894-62955d1f-fc55-487a-9ea1-776e0062e89f.png)

Not sure on the process- you may have to look at @ndmckinley's history, and/or pester them to document it if it isn't intuitive.

Reply:
cl/409490555 should add the documentation to the DCL schema that the template needs to generate the page as it already exists.